server/controllers/manage/premises/placements/departuresController.test.ts

```diff
@@ -527,6 +554,28 @@ describe('DeparturesController', () => {
       expect(errorData).toEqual(expectedErrorData)
     })
 
+    it('returns errors if the move-on category is Tranfer to AP, but the AP is not selected', async () => {
```

Comment:
Slight typo in 'Transfer' -- no biggie but fixing may help with finding relevant code!